### ISSUE TITLE
Update `paint-it-black` theme primary and secondary colors

### ISSUE THREAD

**Issue by robinmolen** [maykinmedia/admin-ui#284]
The primary and secondary colors of the theme "paint it black" are have rather bad color contrast when used for the button components (perhaps other components are in a similarly bad state):

<img width="464" height="121" alt="Image" src="https://github.com/user-attachments/assets/90debd01-76fa-4b6d-98a7-d3c17ddc15c3" />

To solve this contrast issue, we have to update the primary and secondary colors of this theme. Discussed with the designers, and we will use a blue tint similar to the "blue suede shoes" theme.

Important to keep in mind is that the color contrast must be a minimum of 4.5:1. Higher is always better. (The secondary button text and the page background is a nice/important place to test this, but each affected component must keep a minimum color contrast of 4.5:1)

The changes must be checked with the designers to ensure that the theme still looks "nice"

**Comment by robinmolen** [comment]
New dark-mode colors. The success/info/warning/danger variants have been changed to keep consistent with the light-mode theme changes. 
<img width="546" height="113" alt="Image" src="https://github.com/user-attachments/assets/1d99c0c5-d283-4057-97e6-7b9f5a8ded47" />

The designers will continue working on fine-tuning the dark-mode theme, so this might change in the future.
